pkg/kritis/cryptolib/README.md

```diff
@@ -0,0 +1,41 @@
+# Crypto Library
+
+The Crypto Library provides an interface for creating and verifying Attestations through a Signer and Verifier, respectively.
+
+## Components
+
+### Attestation
+
+An Attestation is a statement that a container image meets a policy and can be deployed to a Kubernetes cluster. It contains a signature generated by the private key of a trusted entity (e.g. a build system, vulnerability scanner, etc.). However, as anybody can create an Attestation, it cannot be trusted until it is verified using the entity's corresponding public key. An Attestation is defined independently from the wire format of a signature, allowing the user to store signatures generated by PGP or PKIX keys, or signatures encoded as JWTs.
+
+An Attestation contains a payload and a signature generated over the payload with the trusted entity’s private key. It also contains the ID of the public key which can verify the Attestation’s signature.
+
+### Public Key
```

Comment:
We should consider either 1) inline the spec for the data structures here, or 2) link to the spec in the code.

Reply:
Added a link to the spec.